### PR TITLE
add easyblock for MRtrix

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyblocks/m/mrtrix.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            env.setvar('LD', "%s LDFLAGS OBJECTS -o EXECUTABLE" % os.getenv('CXX'))
+            env.setvar('LDLIB', "%s -shared LDLIB_FLAGS OBJECTS -o LIB" % os.getenv('CXX'))
+            env.setvar('QMAKE_CXX', os.getenv('CXX'))
+            cmd = "python configure -verbose"
+            run_cmd(cmd, log_all=True, simple=True, log_ok=True)
+
+    def build_step(self):
+        """Custom build procedure for MRtrix."""
+        extra_inc_path = os.path.join(get_software_root('GCC'), 'include', 'c++', get_software_version('GCC'))
+        env.setvar('CPATH', ':'.join([extra_inc_path, os.getenv('CPATH', '')]))
```
Comment (boegel) [inline easybuild/easyblocks/m/mrtrix.py]: @wpoely86: doing this fixes this issue for me, based on the info in https://software.intel.com/en-us/forums/intel-c-compiler/topic/338378

```
lib/types.h(107): error: qualified name is not allowed                                                                                                                         
    typedef std::complex<double> cdouble;                                                                                                                                      
...                                                                                                                    
lib/types.h(107): error: expected a ";"                                                                                                                                        
    typedef std::complex<double> cdouble;  
```

It feels like a workaround for a bug though... Are we doing something wrong with `$CPATH` somewhere?

Comment (boegel) [inline easybuild/easyblocks/m/mrtrix.py]: problem was caused by having `include` subdir of `icc` included in `$CPATH`

fixed with https://github.com/wpoely86/easybuild-easyblocks/issues/20 (included in https://github.com/hpcugent/easybuild-easyblocks/pull/756)

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
tested thoroughly with https://github.com/hpcugent/easybuild-easyconfigs/pull/2217 and https://github.com/hpcugent/easybuild-easyconfigs/pull/2218, good to go, thanks for the review @wpoely86!

**Pull request merged**